modules/nf-core/ragtag/patch/tests/main.nf.test

```diff
@@ -0,0 +1,61 @@
+// TODO nf-core: Once you have added the required tests, please run the following command to build this file:
+// nf-core modules test ragtag/patch
+nextflow_process {
+
+    name "Test Process RAGTAG_PATCH"
+    script "../main.nf"
+    process "RAGTAG_PATCH"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "ragtag"
+    tag "ragtag/patch"
+
+    // TODO nf-core: Change the test name preferably indicating the test-data and file-format used
+test("A. thaliana Col-0 test data - ragtag - patch") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'Col-0_2MB' ], // meta map
+                    file('https://raw.githubusercontent.com/nf-core/test-datasets/genomeassembler/A_thaliana_Col-0_2mb/ONT-Col-0_flye_assembly.fasta.gz', checkIfExists: true),
+                    file('https://raw.githubusercontent.com/nf-core/test-datasets/genomeassembler/A_thaliana_Col-0_2mb/Col-CEN_v1.2.Chr1_5MB-7MB.fasta.gz', checkIfExists: true)
```

Comment:
You should'nt use pipeline test data in modules, as you could theoretically move this - the modules branch should be more stable (use the exsiting FASTA files in test-data, or copy these over if necessary)

Reply:
(e.g. just use the SARSCov2 genome twice) - note the output does not need to make biological sense - we just need to keep the tool happily executing without an error.